kitsune/sumo/urls.py

```diff
@@ -1,21 +1,14 @@
-from django.conf.urls import url, include
+from django.conf.urls import url
 from django.views.generic.base import RedirectView
 
 from watchman import views as watchman_views
 
 from kitsune.sumo import views
 
 
-services_patterns = [
-    url("^/monitor$", views.monitor, name="sumo.monitor"),
-    url("^/version$", views.version_check, name="sumo.version"),
-    url("^/error$", views.error, name="sumo.error"),
-]
-
-
 urlpatterns = [
     url(r"^robots.txt$", views.robots, name="robots.txt"),
-    url(r"^services/", include(services_patterns)),
+    url(r"^services/monitor", views.monitor, name="sumo.monitor"),
```

Comment:
We're missing a `$` at the end of the regexp

Reply:
Indeed the url isn't consistent with the old one, which was `example.com/services//monitor`.

Do we actually use this? Can we remove it too?

Reply:
Hopefully we can augment the functionality and have some more consistent reporting. I will update the `$`. The url is not consistent because of the leading slash. This will remove 3 warnings out of the many we have about leading slashes